services/api/.gitignore

```diff
@@ -1,2 +1,3 @@
+.env.example
```

Comment:
you should leave .env in gitignore but commit .env.example.
Anyone who clones the project can create .env localy and not commit it.